docs/extras/adminer.md

```diff
@@ -0,0 +1,3 @@
+If you have `adminer` listed as one of the `installed_extras` inside `config.yml`, you can use Adminer's web-based interface to interact with databases. With Drupal VM running, visit [http://adminer.drupalvm.dev/](http://adminer.drupalvm.dev/), and log in with `drupal` as the username and the password you set in `config.yml` (`drupal_db_password`). Leave the "Server" field blank. The "Database" field is optional.
```

Comment:
This is a duplicate with _Connect to the MySQL database_, but i feel the paragraph isnt long enough to justify a reference, and I do think it's important it's mentioned in both places.